### PR TITLE
[zh-cn]Sync configure-persistent-volume-storage.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Preview:
https://github.com/kubernetes/website/pull/35413/files?short_path=bbbbd45#diff-bbbbd45b0a3a3b296a81ec8be5ba3ee74b6fd68d0f915461289d696a364e5c43

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 1]
```diff
@@ -149,18 +152,33 @@ would provision a network resource like a Google Compute Engine persistent disk,
...
 to set up
-[dynamic provisioning](https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
+[dynamic provisioning](/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
 
 Here is the configuration file for the hostPath PersistentVolume:
 -->
 在生产集群中，你不会使用 hostPath。
 集群管理员会提供网络存储资源，比如 Google Compute Engine 持久盘卷、NFS 共享卷或 Amazon Elastic Block Store 卷。
-集群管理员还可以使用 [StorageClasses](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage) 来设置[动态提供存储](https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes)。
+集群管理员还可以使用 [StorageClasses](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage) 来设置[动态提供存储](/zh-cn/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes)。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:161]: This post doesn't exists.

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 2]
```diff
@@ -215,9 +233,9 @@ Create the PersistentVolumeClaim:
 -->
 创建 PersistentVolumeClaim：
 
-```shell
-kubectl create -f https://k8s.io/examples/pods/storage/pv-claim.yaml
-```
+
+    kubectl apply -f https://k8s.io/examples/pods/storage/pv-claim.yaml
+
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:238]: The `` ```shell `` format is better.

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 3]
```diff
@@ -478,3 +498,5 @@ PersistentVolume are not present on the Pod resource itself.
 * [PersistentVolumeClaimSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#persistentvolumeclaimspec-v1-core)
 
 
+
+
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:502]: nit: Unnecessary newlines.

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 4]
```diff
@@ -51,7 +52,9 @@ do not already have a single-node cluster, you can create one by using
 [Persistent Volumes](/docs/concepts/storage/persistent-volumes/).
 -->
 
-* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置 kubectl 命令行工具以便与集群交互。
+* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置
+
+{{< glossary_tooltip text="kubectl" term_id="kubectl" >}} 命令行工具以便与集群交互。
   如果还没有单节点集群，可以使用
   [Minikube](https://minikube.sigs.k8s.io/docs/) 创建一个。
 .
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:60]: Is the newline intended?

Comment (yanrongshi) [reply content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:60]: I will change it,thanks

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 5]
```diff
@@ -149,18 +150,33 @@ would provision a network resource like a Google Compute Engine persistent disk,
...
+cluster's Node. The configuration also specifies a size of 10 gibibytes and
+an access mode of `ReadWriteOnce`, which means the volume can be mounted as
+read-write by a single Node. It defines the [StorageClass name](/docs/concepts/storage/persistent-volumes/#class)
+`manual` for the PersistentVolume, which will be used to bind
+PersistentVolumeClaim requests to this PersistentVolume.
+-->
+配置文件指定卷位于集群节点上的 `/mnt/data` 路径。
+配置还指定了卷的容量大小为 10 GB，
+访问模式为 `ReadWriteOnce`，
+这意味着该卷可以被单个节点以读写方式安装。w
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:176]: typo...
```suggestion
这意味着该卷可以被单个节点以读写方式安装。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 6]
```diff
@@ -51,7 +52,7 @@ do not already have a single-node cluster, you can create one by using
 [Persistent Volumes](/docs/concepts/storage/persistent-volumes/).
 -->
 
-* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置 kubectl 命令行工具以便与集群交互。
+* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置 {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} 命令行工具以便与集群交互。
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:55]: Long sentences like this mixed Chinese and English, you can break it in the mixed spot, newlines will be parsed as spaces, like below:
```suggestion
* 你需要一个包含单个节点的 Kubernetes 集群，并且必须配置
  {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} 命令行工具以便与集群交互。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 7]
```diff
@@ -149,18 +150,33 @@ would provision a network resource like a Google Compute Engine persistent disk,
...
 to set up
-[dynamic provisioning](https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
+[dynamic provisioning](/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
 
 Here is the configuration file for the hostPath PersistentVolume:
 -->
 在生产集群中，你不会使用 hostPath。
 集群管理员会提供网络存储资源，比如 Google Compute Engine 持久盘卷、NFS 共享卷或 Amazon Elastic Block Store 卷。
-集群管理员还可以使用 [StorageClasses](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage) 来设置[动态提供存储](https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes)。
+集群管理员还可以使用 [StorageClasses](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage) 来设置[动态提供存储](/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes)。
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:159]: ditto.
```suggestion
集群管理员还可以使用
[StorageClasses](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage)
来设置[动态提供存储](/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes)。
```

File: content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 8]
```diff
@@ -149,18 +150,33 @@ would provision a network resource like a Google Compute Engine persistent disk,
...
+an access mode of `ReadWriteOnce`, which means the volume can be mounted as
+read-write by a single Node. It defines the [StorageClass name](/docs/concepts/storage/persistent-volumes/#class)
+`manual` for the PersistentVolume, which will be used to bind
+PersistentVolumeClaim requests to this PersistentVolume.
+-->
+配置文件指定卷位于集群节点上的 `/mnt/data` 路径。
+配置还指定了卷的容量大小为 10 GB，
+访问模式为 `ReadWriteOnce`，
+这意味着该卷可以被单个节点以读写方式安装。w
+配置文件还在 PersistentVolume 中定义了 [StorageClass 的名称](/zh-cn/docs/concepts/storage/persistent-volumes/#class)
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:177]:
```suggestion
配置文件还在 PersistentVolume 中定义了
[StorageClass 的名称](/zh-cn/docs/concepts/storage/persistent-volumes/#class)
```

**Review by Sea-n — COMMENTED** [review]
Thanks, it's ready for merge! 🎉
/lgtm

**Pull request merged**